CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## [v1.4.3](http://github.com/ndustrialio/contxt-sdk-js/tree/v1.4.3) (2019-06-25)
+
+**Added**
+
+-Added `Users#subscribeEvent` for subscribing an event to a user
```

Comment:
The descriptions on these are backwards -- it's subscribing a user to an event

Reply:
👍